test/unit/alpha.js

```diff
@@ -76,4 +75,30 @@ describe('Alpha transparency', function() {
       });
   });
 
+  it('Enlargement with non-nearest neighbor interpolation shouldn’t cause dark edges', function(done) {
+    var BASE_NAME = 'alpha-premultiply-enlargement-2048x1536-paper.png';
+    var actual = fixtures.path('output.' + BASE_NAME);
+    var expected = fixtures.expected(BASE_NAME);
+    sharp(fixtures.inputPngAlphaPremultiplicationSmall)
+      .resize(2048, 1536)
+      .interpolateWith('bicubic')
+      .toFile(actual, function(err) {
+        if (err) throw err;
+        fixtures.assertEqual(expected, actual, done);
+      });
+  });
+
+  it('Reduction with non-nearest neighbor interpolation shouldn’t cause dark edges', function(done) {
+    var BASE_NAME = 'alpha-premultiply-reduction-1024x768-paper.png';
+    var actual = fixtures.path('output.' + BASE_NAME);
+    var expected = fixtures.expected(BASE_NAME);
+    sharp(fixtures.inputPngAlphaPremultiplicationLarge)
+      .resize(1024, 768)
+      .interpolateWith('bicubic')
+      .toFile(actual, function(err) {
+        if (err) throw err;
+        fixtures.assertEqual(expected, actual, done);
```

Comment:
The Windows CI is reporting `Expected images be equal. Mean squared error: 0.0003159345069434494.` so you'll probably need to allow a small MSE value via a threshold or similar.